concepts/variadic-functions/about.md

```diff
@@ -0,0 +1,57 @@
+# About
+Functions in general accept only a fixed number of arguments. A variadic function is a function that accepts a variable number of arguments. If the last parameter of a function definition is prefixed by ellipsis ..., then the function can accept any number of arguments for that parameter.
+
+```go
+func hello(a int, b ...int) {
+}
+```
+
+In the above function, parameter b is variadic and we can pass 0 or more arguments to b.
+
+```go
+    func hello(5, 6){}
+    func hello(5, 6, 7){}
+    func hello(5){}
```

Comment:
There should be no `func` and `{}` for these as you want to show the function call.

Reply:
resolved